include/BulkMove.php

```diff
@@ -171,8 +171,8 @@ private function load_ajax_handlers() {
 	private function get_ajax_handlers() {
 		if ( empty( $this->ajax_handlers ) ) {
 			$this->ajax_handlers[] = new BM_Request_CustomTaxonomyAction();
+			$this->ajax_handlers[] = new BM_Request_LoadTaxonomyTerm();
 		}
-
```

Comment:
Why this blank line was removed?

Reply:
@rajanit2000 This comment is not addressed.

Reply:
@sudar it's by mistake i have revert it

Reply:
@rajanit2000 

Looks like you added blank line at the wrong place.

Reply:
@sudar If I add blank space its comes StyleCI error. How can I sort out this

Reply:
@rajanit2000 Okay. Let's leave it for now.